<ui version="4.0" >
 <class>patchExchange_form</class>
 <widget class="QWidget" name="patchExchange_form" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_48" >
   <property name="geometry" >
    <rect>
     <x>350</x>
     <y>20</y>
     <width>301</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title" >
    <string>Results</string>
   </property>
   <widget class="QListWidget" name="results_listWidget" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>30</y>
      <width>256</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_49" >
   <property name="geometry" >
    <rect>
     <x>690</x>
     <y>20</y>
     <width>301</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title" >
    <string>Details</string>
   </property>
   <widget class="QLabel" name="label_119" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>30</y>
      <width>57</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_120" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>50</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Contributor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_121" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>70</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Originator:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_122" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>90</y>
      <width>57</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_123" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>110</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Description:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_124" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>250</y>
      <width>57</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Tags:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_125" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>270</y>
      <width>57</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Added:</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_label" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>30</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="margin" >
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="contributor_label" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>50</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="margin" >
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="originator_label" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>70</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="margin" >
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="type_label" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>90</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="margin" >
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="desc_label" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>130</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap" >
     <bool>true</bool>
    </property>
    <property name="margin" >
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="tags_label" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>250</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="margin" >
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="added_label" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>270</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="margin" >
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="epxCopy_pushButton" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>200</x>
      <y>310</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>&amp;Copy</string>
    </property>
   </widget>
   <widget class="QLabel" name="private_label" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>110</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text" >
     <string>public</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="epxDelete_pushButton" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>310</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="palette" >
     <palette>
      <active>
       <colorrole role="Button" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text" >
     <string>&amp;Delete</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_47" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>20</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title" >
    <string>Query</string>
   </property>
   <widget class="QComboBox" name="type_comboBox" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>30</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text" >
      <string>All</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_115" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>30</y>
      <width>57</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_116" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Added in the Last</string>
    </property>
    <property name="wordWrap" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="since_comboBox" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>70</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="currentIndex" >
     <number>0</number>
    </property>
    <item>
     <property name="text" >
      <string>All</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>1 day</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>7 days</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>1 month</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>3 months</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>1 year</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_117" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>120</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Contributor</string>
    </property>
   </widget>
   <widget class="QComboBox" name="contributor_comboBox" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>120</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text" >
      <string>All</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="epxQuery_pushButton" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>260</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>&amp;Query</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_118" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>220</y>
      <width>57</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Tags</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tags_lineEdit" >
    <property name="geometry" >
     <rect>
      <x>60</x>
      <y>220</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="origin_comboBox" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>170</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text" >
      <string>All</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_126" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>170</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Origin</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="stats_groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>330</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title" >
    <string>Statistics</string>
   </property>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Registered Users:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Patches contributed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Contributors:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>80</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Origins:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>100</y>
      <width>115</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Last Access (UTC):</string>
    </property>
   </widget>
   <widget class="QLabel" name="users_label" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>20</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>0</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="patches_label" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>40</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>0</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="contributors_label" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>60</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>0</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="origins_label" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>80</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>0</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="access_label" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>100</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>0</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
